resource.html

```diff
@@ -859,7 +859,7 @@ <h3 class="hover-underline-animation">Data Structure and Algorithm Tutorials</h3
             <br><br>
         </section>
 
-        <center>
+        
```

Comment:
Revert this, this should be centered aligned...

Reply:
I reverted Back Center Tag in resource.html File

Reply:
Thanks looks good...